eks-service-definition.yml

```diff
@@ -30,6 +30,10 @@ provision:
       pattern: ^[a-z0-9][a-z0-9-]*[a-z0-9]$
     default: ${str.truncate(64, "${request.instance_id}")}
     details: A subdomain to use for the cluster instance. Default is the instance ID. Make sure the name meets AWS requirements (https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/DomainNameFormat.html)
+  - field_name: use_hardened_ami
+    type: boolean
+    required: false
+    details: "Specify whether we want to use the GSA hardened AMI."
```

Comment:
This is good, but the tests will fail because there's one more block missing under `computed_inputs` (somewhere around line 100 of this file).

Example,
```terraform
  - name: use_hardened_ami
    type: boolean
    default: false
    overwrite: true
```

Reply:
No need to be sorry! I am learning a lot through failure 😆 Thanks for all your help.

Reply:
It is a bit mysterious.. there wasn't a good way for me to debug it here, so I cloned your fork and was able to start it by setting all of the required environment variables and such.  Since it works in isolation, I suspect it's because the PR is coming from a fork.  

The part that is error-ing is the `make up` command which is supposed to start the CSB with the eks-brokerpak as a plugin.  A pre-cursor to that is the creation of the `.env.secrets` file.  As I mentioned earlier, I was able to set this up and run it locally on your fork.  My suspicion is that the github secrets that set this up for the github action aren't shared to your run as "fork run" if that makes sense.  Specifically, this part is failing without error, https://github.com/GSA/datagov-brokerpak-eks/runs/6237381422?check_suite_focus=true#step:3:1

@mogul Can you share some insight into whether you think I'm right or wrong?